marvel_superheroes/src/components/HomePage/HomePage.tsx

```diff
@@ -17,41 +19,40 @@ export interface IHero {
     path: string
 }
 
-type IProps = RouteComponentProps;
+export type IHeroProps = RouteComponentProps & IGetHeroes;
 
-export interface IGetHeroes {
-    offset: number,
+interface IGetHeroes {
+    heroes: IHero[],
+    totalOfItems: number,
     nameStartsWith: string,
     orderBy: string,
-    limit: number,
     progressBar: boolean,
+    currentHeroesPage: number,
+    setHeroesNameStartWith(nameStartsWith: string): ISetHeroesNameStartWith,
+    setHeroesOrderBy(orderBy: string): ISetHeroesOrderBy,
```

Comment:
all the same about this interface, no need to pass all methods and state to redux. There is no sense in moving setHeroesNameStartWith, setHeroesOrderBy, setHeroesCurrentPage and so on